.github/workflows/command-block.yml

```diff
@@ -4,9 +4,16 @@ on:
     paths:
       - 'commands/docs/**'
 
+# Fix Resource not accessible by integration (updatePullRequest)
+permissions:
+  contents: write
+  pull-requests: write
+
 jobs:
   comment:
     runs-on: ubuntu-latest
+    # If the docs was updated by hustcer, skip the commenting step
+    if: ${{!contains(github.event.pull_request.user.login, 'hustcer') }}
```

Comment:
Not security critical here but that would match all user names containing yours as a substring. either a direct match or an array of user names in the first arg of `contains` would make sense.